proto/spire/next/api/bundle/v1/bundle.proto

```diff
@@ -6,10 +6,36 @@ import "google/protobuf/empty.proto";
 import "spire/next/types/bundle.proto";
 
 service Bundle {
+    // Lists bundles.
     rpc ListBundles(ListBundlesRequest) returns (ListBundlesResponse);
+
+    // Gets a bundle. If the bundle does not exist, NOT_FOUND is returned.
     rpc GetBundle(GetBundleRequest) returns (spire.types.Bundle);
+
+    // Creates a bundle. If the bundle already exists, ALREADY_EXISTS is returned.
+    rpc CreateBundle(CreateBundleRequest) returns (spire.types.Bundle);
+
+    // Batch creates one or more bundles.
+    rpc BatchCreateBundle(BatchCreateBundleRequest) returns (BatchCreateBundleResponse);
+
+    // Updates a bundle. If the bundle does not exist, NOT_FOUND is returned.
+    rpc UpdateBundle(UpdateBundleRequest) returns (spire.types.Bundle);
+
+    // Batch updates one or more bundles.
+    rpc BatchUpdateBundle(BatchUpdateBundleRequest) returns (BatchUpdateBundleResponse);
```

Comment:
What about batch create? If there is a use for batch create, I could see at least see a use for batch set (which is the RPC I'd probably choose instead of batch create because I probably wouldn't care about the existing bundles).